mage_ai/ai/constants.py

```diff
@@ -1,7 +1,14 @@
-from enum import Enum
+try:
```

Comment:
since the code can be duplicated across multiple files? i would suggest putting it in the shared/enum.py and import the StrEnum from shared.enum

Reply:
Sure, made the suggested change.